crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -143,7 +143,7 @@ def adminrepocheck
             repos: {}
           }
           ret[:openstack][:repos][admin_architecture.to_sym] = {
-            missing: ["SUSE OpenStack Cloud 7"]
+            missing: ["SUSE-OpenStack-Cloud-7"]
```

Comment:
`["SUSE-OpenStack-Cloud-7-Pool", "SUSE-OpenStack-Cloud-7-Updates"]`

@skazi0 , is this what UI expects?